examples/debugging/simple_spread/feedforward/decentralised/run_maddpg_scaled.py

```diff
@@ -0,0 +1,136 @@
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Example running feedforward MADDPG on debug MPE environments."""
+
+import functools
+from datetime import datetime
+from typing import Any
+
+import launchpad as lp
+import sonnet as snt
+from absl import app, flags
+from launchpad.nodes.python.local_multi_processing import PythonProcess
+
+from mava.systems.tf import maddpg_scaled
+from mava.systems.tf.maddpg import make_default_networks
+from mava.utils import lp_utils
+from mava.utils.environments import debugging_utils
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string(
+    "env_name",
+    "simple_spread",
+    "Debugging environment name (str).",
+)
+flags.DEFINE_string(
+    "action_space",
+    "continuous",
+    "Environment action space type (str).",
+)
+flags.DEFINE_string(
+    "mava_id",
+    str(datetime.now()),
+    "Experiment identifier that can be used to continue experiments.",
+)
+flags.DEFINE_string("base_dir", "~/mava/", "Base dir to store experiments.")
+
+
+def main(_: Any) -> None:
+
+    # environment
+    environment_factory = functools.partial(
+        debugging_utils.make_environment,
+        env_name=FLAGS.env_name,
+        action_space=FLAGS.action_space,
+    )
+
+    # networks
+    network_factory = lp_utils.partial_kwargs(make_default_networks)
+
+    # loggers
+    # set custom logger config for each process
+    # -- log trainer,executor and evaluator to TF
+    # -- log only evaluator to terminal
+    log_every = 10
+    logger_config = {
+        "trainer": {
+            "directory": FLAGS.base_dir,
+            "to_terminal": False,
+            "to_tensorboard": True,
+            "time_stamp": FLAGS.mava_id,
+            "time_delta": log_every,
+        },
+        "executor": {
+            "directory": FLAGS.base_dir,
+            "to_terminal": False,
+            "to_tensorboard": True,
+            "time_stamp": FLAGS.mava_id,
+            "time_delta": log_every,
+        },
+        "evaluator": {
+            "directory": FLAGS.base_dir,
+            "to_terminal": True,
+            "to_tensorboard": True,
+            "time_stamp": FLAGS.mava_id,
+            "time_delta": log_every,
+        },
+    }
+
+    # Checkpointer appends "Checkpoints" to checkpoint_dir
+    checkpoint_dir = f"{FLAGS.base_dir}/{FLAGS.mava_id}"
+
+    # distributed program
+    program = maddpg_scaled.MADDPG(
+        environment_factory=environment_factory,
+        network_factory=network_factory,
+        logger_config=logger_config,
+        num_executors=2,
+        num_trainers=3,
+        shared_weights=False,
+        trainer_net_config={
```

Comment:
Do we have default trainer net configs as well? For example, `{"trainer_0": ["agent_0", "agent_1", "agent_2"]}`? Also, the argument doesn't seem to be in the system code?

Reply:
I added a default setup for the trainer_config :+1: So if no trainer_net_config is specified it sets trainer_0 to train on all the networks. We don't inherit from the system as it is still used by other algorithms which do not have the trainer_net_config setup. Does this make sense or can we do something else about this?